core/src/com/unciv/ui/pickerscreens/ImprovementPickerScreen.kt

```diff
@@ -109,8 +126,24 @@ class ImprovementPickerScreen(
                     && improvement.name != Constants.cancelImprovementOrder)
             if (tileInfo.improvement != null && removeImprovement) labelText += "\n" + "Replaces [${tileInfo.improvement}]".tr()
 
-            val pickNow = when {
-                suggestRemoval -> "${Constants.remove}[${tileInfo.getLastTerrain().name}] first".toLabel()
+            var proposedSolutions = mutableListOf<String>()
+
+            if (suggestRemoval) proposedSolutions.add("${Constants.remove}[${tileInfo.getLastTerrain().name}] first")
```

Comment:
Most of the proposed solutions have a .tr() at the end, this does not.
Also, there isn't any space between the 'remove' and the terrain name - how does this look?
Also also, we *could* move the .tr() into the joinToString below like `.joinToString("\n"){it.tr()}`, not necessary but nice IMO

Reply:
Solved by SomeTroglodyte.

Reply:
> Also, there isn't any space between the 'remove' and the terrain name - how does this look?

This constant has value "Remove ", with a trailing space. I didn't add this string, it was already there. FWIW, I think it should rather inline the constant value instead of referring to it, but I'd rather not touch it in this PR, it's not about that.

Reply:
> with a trailing space

A thing I already knew but _looked up again_ while checking this. Worth thinking about some day. But IMHO we should look for a solution that does _not_ depend on the names in the long run (the Constant exists because elsewhere it's used for a startsWith). Just FYI. Well done.